rotkehlchen/chain/bitcoin/utils.py

```diff
@@ -39,7 +43,8 @@ def is_valid_btc_address(value: str) -> bool:
     return (
         is_valid_base58_address(value) or
         is_valid_bech32_address(value) or
-        is_valid_bech32_bip350_address(value)
+        is_valid_bech32_bip350_address(value) or
+        is_valid_bitcoin_cash_address(value)
```

Comment:
is this true? all the valid bch addresses are valid btc addresses? The set of addresses is not different between both of them (until the split sure it is valid)

Reply:
right, they are not the same